<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cdialog</class>
 <widget class="QDialog" name="Cdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>课程信息</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="Cid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cpoint">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>学分</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cterm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>学期</string>
   </property>
  </widget>
  <widget class="QLabel" name="C_room">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>教室</string>
   </property>
  </widget>
  <widget class="QLabel" name="C_socre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>成绩</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Cid">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Cpoint">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Cterm">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Croom">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Cscore">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Cname">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Cdialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Cdialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
